### ISSUE TITLE
Skia cannot find the skcms module

### ISSUE THREAD

**Issue by X1nto** [microsoft/vcpkg#27219]
**Describe the bug**
Skia's skcms (and other modules) cannot be found.

**Environment**
- OS: Windows 11 22H2 (22621.674)
- Compiler: Irrelevant

**To Reproduce**
1. ./vcpkg install skia:x64-windows-static
2. ./vcpkg install glfw3:x64-windows-static
3. Make a Skia project
4. `#include "skia/core/SkColorSpace.h"`
5. See the error

Reproduction code (requires glfw3 and skia[gl]):
```cpp

#include "skia/core/SkRRect.h"
#include "skia/core/SkGraphics.h"
#include "skia/core/SkPaint.h"
#include "skia/core/SkCanvas.h"
#include "skia/core/SkSurface.h"
#include "skia/core/SkColorSpace.h"
#include "skia/gpu/gl/GrGLInterface.h"
#include "skia/gpu/GrDirectContext.h"
#include "GLFW/glfw3.h"

int main()
{
	GLFWwindow* window;
	if (!glfwInit())
	{
		return -1;
	}
	window = glfwCreateWindow(1280, 720, "sample", nullptr, nullptr);

	auto interface = GrGLMakeNativeInterface();
	GrDirectContext* context = GrDirectContext::MakeGL(interface).release();

	GrGLFramebufferInfo framebufferInfo;
	framebufferInfo.fFBOID = 0;
	framebufferInfo.fFormat = GL_RGBA8;

	SkColorType colorType = kRGBA_8888_SkColorType;
	GrBackendRenderTarget backendRenderTarget(1280, 720, 0, 0, framebufferInfo);

	SkSurface* surface = SkSurface::MakeFromBackendRenderTarget(context, backendRenderTarget, kBottomLeft_GrSurfaceOrigin, colorType, nullptr, nullptr).release();
	if (surface == nullptr)
	{
		abort();
	}

	while (!glfwWindowShouldClose(window))
	{
		glfwWaitEvents();
		SkPaint paint;
		paint.setColor(SK_ColorRED);
		surface->getCanvas()->drawRRect(SkRRect(), paint);
	}

	glfwDestroyWindow(window);
	glfwTerminate();
	return 0;
}
``` 
```cmake
find_package(glfw3 CONFIG REQUIRED)
find_package(skia CONFIG REQUIRED)
target_link_libraries(Project PRIVATE glfw skia skia::skia)
```

**Expected behavior**
The build should succeed and a GL window should appear.

**Failure logs**
None.

**Additional context**
There was an [issue on Google's groups](https://groups.google.com/g/skia-discuss/c/RAs9yedISpM) regarding the same problem, but the solution there cannot be applied here.

**Comment by FrankXie05** [comment]
@X1nto Thanks for posting this issue, it will be fixed by PR #26876 :)

**Comment by X1nto** [comment]
@FrankXie05 Thank you!
